src/Microsoft.AspNet.OData.Shared/DeltaOfTStructuralType.cs

```diff
@@ -673,5 +678,18 @@ private bool TrySetNestedResourceInternal(string name, object deltaNestedResourc
 
             return true;
         }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <returns></returns>
+        public Microsoft.OData.Edm.IEdmTypeReference GetEdmType()
```

Comment:
>GetEdmType [](start = 53, length = 10)

If we don't think this will be used, we should probably put it in an explicit implementation - i.e.,

Microsoft.OData.Edm.IEdmTypeReference IEdmObject.GetEdmType()

Problem is, Delta<TStructuralType> is agnostic of a model, therefore has no EdmType.

Is this a problem in any scenarios? Do we need a way to set an IEdmTypeReference (or model) on the Delta<TStructuralType> object?

Reply:
Made explicit interface.

---
In reply to: [518403225](https://github.com/OData/WebApi/pull/2275#discussion_r518403225) [](ancestors = 518403225)